### DISCUSSION TITLE
..

### DISCUSSION THREAD

**Discussion by theonewhoknocks878** [saarock/notty#25] [General]

**Comment by saarock** [comment]
NO just at rest

**Comment by theonewhoknocks878** [comment]
Why are you at rest @saarock ?

**Comment by theonewhoknocks878** [comment]
if you want, [Slack Invite to VisiUI](https://join.slack.com/t/visiui/shared_invite/zt-2oge4h1vn-J18RupLnaPKbmgF_AnhuSw)
@saarock
